src/rgw/driver/d4n/d4n_policy.cc

```diff
@@ -591,16 +591,16 @@ void LFUDAPolicy::cleaning(const DoutPrefixProvider* dpp)
       block.size = 0;
       block.blockID = 0;
       if (c_obj->have_instance()) {
-        dir->get(dpp, &block, null_yield);
+        blockDir->get(dpp, &block, null_yield);
```

Comment:
why did this error happen in the first place? It built fine at my end and I tested things before pushing the last commit.

Reply:
This wasn't an error. I added an `objDir` variable because I had to update the object in the `cleaning` process so I just wanted to differentiate both directory instances more clearly.